client/cypress/integration/query/parameter_spec.js

```diff
@@ -286,11 +295,24 @@ describe('Parameter', () => {
       cy.getByTestId('ParameterApplyButton')
         .click();
 
-      cy.get('@Input').then(($input) => {
-        const now = Cypress.moment($input.val(), 'DD/MM/YY HH:mm');
-        cy.getByTestId('DynamicTable')
-          .should('contain', now.format('YYYY-MM-01 HH:mm'));
-      });
+      cy.getByTestId('DynamicTable')
+        .should('contain', Cypress.moment(this.now).format('YYYY-MM-DD HH:mm'));
+    });
+
+    it('allows picking a dynamic date', function () {
+      cy.getByTestId('DynamicButton')
+        .click()
+        .click();
```

Comment:
Assuming the double click is the picker workaround.

```suggestion
        .click(); // workaround for datepicker display bug
```